### PR TITLE
fix: Improve external links for 7.32.x

### PR DESCRIPTION
**Pull request by max-cx**
## What does this pull request change?
Improving the docs UX by providing topic-specific external links and relocating the links within the text according to their relevance.

## What issues does this pull request fix or reference?
2974

## Specify the version of the product this pull request applies to
7.32

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by tolusha — APPROVED** [review]

File: modules/administration-guide/partials/con_external-database-setup.adoc [hunk 1]
```diff
@@ -34,13 +34,14 @@ The recommended configuration of the example database is:
 |Automated backups | enabled (30 days)
 |===
 
-include::partial$proc_configuring-external-PostgreSQL.adoc[leveloffset=+1]
+.Additional resources
```
Comment (themr0c) [inline modules/administration-guide/partials/con_external-database-setup.adoc:37]: Additional resources should be the last section in a mod docs module. They can't be followed by include statements.

File: modules/administration-guide/partials/con_external-database-setup.adoc [hunk 2]
```diff
@@ -34,13 +34,14 @@ The recommended configuration of the example database is:
...
+* link:https://postgresql.org/docs/current/[PostgreSQL Documentation]
+* link:https://postgresql.org/docs/current/backup.html[PostgreSQL Documentation - Backup and Restore]
+* link:https://aws.amazon.com/rds/[Amazon RDS]
 
-.Additional resources
+include::partial$proc_configuring-external-PostgreSQL.adoc[leveloffset=+1]
 
-* link:https://www.postgresql.org/[PostgreSQL]
-* link:https://aws.amazon.com/rds/[RDS]
+include::partial$proc_configuring-che-to-work-with-external-PostgreSQL.adoc[leveloffset=+1]
```
Comment (themr0c) [inline modules/administration-guide/partials/con_external-database-setup.adoc:45]: A concept module is not supposed to include procedures, can you transform the files schemes to have 
* a parent assembly including
** a concept
** 2 procedures

Then the additional resources will fit in the desired place in the concept.

File: modules/administration-guide/partials/con_persistent-volumes-backups.adoc [hunk 3]
```diff
@@ -28,6 +28,7 @@ NOTE: Alternatively, you can use backup solutions dependent on the underlying st
 * link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[Persistent Volumes documentation]
 * link:https://www.gluster.org/[Gluster documentation]
 * link:https://docs.ceph.com/docs/master/[Ceph documentation]
-* link:https://github.com/vmware-tanzu/velero[Velero on GitHub]
+* link:https://velero.io/docs/main/disaster-case/[Velero Docs - Disaster Recovery]
+* link:https://velero.io/docs/main/restore-reference/[Velero Docs - Restore Reference]
```
Comment (themr0c) [inline modules/administration-guide/partials/con_persistent-volumes-backups.adoc:32]:
```suggestion
* link:https://velero.io/docs/main/disaster-case/[Velero documentation - Disaster Recovery]
* link:https://velero.io/docs/main/restore-reference/[Velero documentation - Restore Reference]
```
for consistency with the lines just above

File: modules/administration-guide/partials/con_persistent-volumes-backups.adoc [hunk 4]
```diff
@@ -28,6 +28,7 @@ NOTE: Alternatively, you can use backup solutions dependent on the underlying st
 * link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[Persistent Volumes documentation]
```
Comment (max-cx) [inline modules/administration-guide/partials/con_persistent-volumes-backups.adoc]:
```suggestion
* link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[Kubernetes Documentation - Persistent Volumes documentation]
```

File: modules/administration-guide/partials/con_persistent-volumes-backups.adoc [hunk 5]
```diff
@@ -28,6 +28,7 @@ NOTE: Alternatively, you can use backup solutions dependent on the underlying st
...
 * link:https://www.gluster.org/[Gluster documentation]
```
Comment (max-cx) [inline modules/administration-guide/partials/con_persistent-volumes-backups.adoc]:
```suggestion
* link:https://docs.gluster.org/en/latest/Quick-Start-Guide/Quickstart/[Gluster Docs]
```

File: modules/administration-guide/partials/con_persistent-volumes-backups.adoc [hunk 6]
```diff
@@ -28,6 +28,7 @@ NOTE: Alternatively, you can use backup solutions dependent on the underlying st
...
 * link:https://docs.ceph.com/docs/master/[Ceph documentation]
```
Comment (max-cx) [inline modules/administration-guide/partials/con_persistent-volumes-backups.adoc]:
```suggestion
* link:https://docs.ceph.com/docs/master/[Ceph Documentation]
```

File: modules/administration-guide/partials/con_external-database-setup.adoc (hunk 1 above)
Comment (max-cx) [reply modules/administration-guide/partials/con_external-database-setup.adoc:37]: @themr0c, :) I don't understand, can you please clarify why additional resources can't be followed by include statements?

File: modules/administration-guide/partials/con_external-database-setup.adoc (hunk 2 above)
Comment (max-cx) [reply modules/administration-guide/partials/con_external-database-setup.adoc:45]: @themr0c I get it. Sure. Thanks. BTW, I'll do the rest of my review of these files in a different topic branch.

**Comment by max-cx** [comment]
Work on this branch continues in PR https://github.com/eclipse/che-docs/pull/2121/

**Pull request closed without merging**